native-jfr-demo/README.md

```diff
@@ -0,0 +1,73 @@
+# Native Executables with JFR Example
+
+This example demonstrates how JDK Flight Recorder (JFR) as a tool for collecting diagnostic and profiling data running Java application, built into a JVM. 
+GraalVM Native Image supports JFR events and users can use [`jdk.jfr.Event`](https://docs.oracle.com/en/java/javase/11/docs/api/jdk.jfr/jdk/jfr/Event.html) API with a similar experience to using JFR in the Java HotSpot VM.
+
+To record JFR events when running a native executable, JFR support and JFR recording must be enabled, and this guide covers how to do that.
+
+> Note: JFR events recording is not supported on GraalVM JDK for Windows. JFR is only supported with native executables built on GraalVM JDK 11.
```

Comment:
```suggestion
> Note: JFR event recording is not supported on GraalVM JDK for Windows. JFR is only supported with native executables built on GraalVM JDK 11.
```
I'm not sure I understand what this means.
Is JFR with GraalVM JDK11 supported on Windows?